spec/controllers/global_user_roles_controller_spec.rb

```diff
@@ -0,0 +1,142 @@
+require "rails_helper"
+
+RSpec.describe GlobalUserRolesController do
+  shared_examples_for "the user must log in" do
```

Comment:
Sounds good to me since it's a step toward phasing out `controller_spec_helper.rb`.